docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started
+In this tutorial, you build your own Obscuro dApp from the start. This is an excellent way to experience a typical development process as you learn how Obscuro dApps are built, design concepts, tools and terminology.
+
+# Let's go
+This 'Guessing Game' tutorial provides an introduction to the fundamentals of Obscuro and shows you how to:
+- Set up your local Ethereum development environment for Obscuro.
+- Use Obscuro concepts to develop a dApp
+
+What is the *Guessing Game*?
+
+The Guessing Game is a dApp that you will build that demonstrates a basic Obscuro use case, a simple number guessing game. The contract generates a random secret number, which is never revealed to anyone, including node operators, developers or end-user because of the privacy provided by Obscuro. The goal of the game is to guess this number, and each time an attempt is made, an entrance fee of 1 token is paid. If a user correctly guesses the number, the contract will pay out all of the accumulated entrance fees to them and reset itself with a new random number.
+
+Without Obscuro, it would be possible to look up the internal state of the contract and cheat, and the game wouldn't work.
+
+The dApp has many of the features that you'd expect to find, including:
+- Generating a random number known only to the dApp
+- Allow users to guess the numbers by depositing a token for each play
+- Colder/Warmer functionality to help guide users to the correct number
+- Events to alert users whether they have won or not
+- Track all user's plays
+- Contract owner privileges to start and stop the game
+
+# Set up your environment
```

Comment:
I'll try out the tutorial tomorrow.

I was wondering if we could just take an existing Ethereum contract (ERC20 or something) and add privacy to that, to make a simpler demo?

Reply:
Good idea. I've added a ticket and will add that as another tutorial/sample